master/internal/db/postgres_experiments.go

```diff
@@ -1156,9 +1156,11 @@ func (db *PgDB) ExperimentByID(id int) (*model.Experiment, error) {
 	var experiment model.Experiment
 
 	if err := db.query(`
-SELECT id, state, config, model_definition, start_time, end_time, archived,
-	   git_remote, git_commit, git_committer, git_commit_date, owner_id, job_id
-FROM experiments
+SELECT e.id, state, config, model_definition, start_time, end_time, archived,
+	   git_remote, git_commit, git_committer, git_commit_date, owner_id, notes,
+		 job_id, u.username as username
```

Comment:
what's the purpose for the additional selection of nodes and username?

Reply:
in general, I think wherever we're returning the experiment model we want to fetch all the attributes it defines unless otherwise is clearly requested. Here `notes` was missing and I just added `username`